perf/benchmark/run_benchmark_job.sh

```diff
@@ -174,6 +174,9 @@ DEFAULT_CR_PATH="${ROOT}/istio-install/istioctl_profiles/default.yaml"
 CONFIG_DIR="${WD}/configs/istio"
 
 for dir in "${CONFIG_DIR}"/*; do
+    if [[ "${dir}" =~ "mixer" ]];then
```

Comment:
I would suggest creating a config file like: run_with_pipeline.config, where you can define what config groups to run with the pipeline easily. It would be better than hardcoding in the code. And it is easy to update

for example
```
mixer: disabled
telemetryv2_stats: enabled
telemetryv2_sd: enabled
```

Reply:
good point! thanks